pkg/3scale/amp/operator/zync_options_provider.go

```diff
@@ -254,6 +260,12 @@ func (z *ZyncOptionsProvider) zyncQuePodTemplateLabels() map[string]string {
 		labels[k] = v
 	}
 
+	if z.apimanager.Spec.Zync.QueSpec.Labels != nil {
```

Comment:
this `if` is not needed

Reply:
Addressed in this [commit](https://github.com/3scale/3scale-operator/pull/835/commits/f903bf41523372ad2297594e4efac295a12c0665)